src/report-command/language-files.ts

```diff
@@ -43,16 +43,19 @@ function extractI18nItemsFromLanguageFiles (languageFiles: SimpleFile[]): I18NLa
   return languageFiles.reduce((accumulator, file) => {
     const language = file.fileName.substring(file.fileName.lastIndexOf('/') + 1, file.fileName.lastIndexOf('.'));
 
+    if (!accumulator[language]) {
```

Comment:
Could we avoid this check using the initial value?

Reply:
How ? Because in the initial value of the reduce `language` is not defined and is not known
Or I can't see, and I would like to know how you would do that?